.github/workflows/create-pr-on-tags.yaml

```diff
@@ -14,7 +14,7 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - name: Set GH_REF
-        run: echo "GH_REF=`echo $(git fetch -t -q | sort --version-sort | tail -n1)`" >> $GITHUB_ENV
+        run: echo "GH_REF=`echo $(git fetch -t -q && git describe --tags)`" >> $GITHUB_ENV
```

Comment:
Does this always do what you want when there are multiple tags at head? (Is that something that can happen here?)

Reply:
The same code is being used for datagovuk_publish, and that appears to be working well

Reply:
ok, I've found a better one which I've just tested on my machine